src/_data/sectionAbout.json

```diff
@@ -48,5 +48,30 @@
                 "body": "Plan your trip and create lists of your favorite places"
             }
         ]
+    },
+    "ch": {
```

Comment:
Do we use simplified Chinese here?
If so, let's put `zh-Hans` as lang code.
If we're not sure, let's use `zh`.

P.S. We use simplified chinese for mobile app.
<img width="303" alt="Screenshot 2022-08-19 at 15 53 08" src="https://user-images.githubusercontent.com/7137128/185622659-5cd17cbb-6d1b-4db4-a1dc-f1133e46240e.png">